config/webpack/development.js

```diff
@@ -15,15 +15,10 @@ const customRules = {
       {
         test: /\.jsx?|.es6?|.spec.js?$/,
         exclude: /(node_modules)/,
-        use: [
-          {
-            loader: 'babel-loader',
-            options: {
-              presets: ['es2015', 'react'],
-              plugins: ['transform-object-rest-spread', 'transform-class-properties', 'transform-es2015-destructuring']
-            }
-          }
-        ]
+        loader: 'babel-loader',
```

Comment:
Why removing the `use`?
Will babel be the only loader?

Reply:
use is now defined in `.babelrc`.